operations/utility_storage/Troubleshoot_Ceph_New_RGW_Deployment_Failing.md

```diff
@@ -0,0 +1,149 @@
+            pgs:     625 active+clean
+
+        io:
+            client:   8.4 KiB/s rd, 1.8 MiB/s wr, 5 op/s rd, 176 op/s wr
+
+        progress:
+            Updating rgw.site1.zone1 deployment (+2 -> 2) (2s)
+            [============================]
+
+
+        2024-09-20T19:04:56.337199+0000 mgr.ncn-s002.mneqjy [INF] Removing key for client.rgw.site1.zone1.ncn-s002.mgujxl
+        2024-09-20T19:04:56.371965+0000 mgr.ncn-s002.mneqjy [ERR] Failed while placing rgw.site1.zone1.ncn-s002.mgujxl on ncn-s002: cephadm exited with an error code: 1, stderr: Non-zero exit code 125 from /usr/bin/podman container inspect --format {{.State.Status}} ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw-site1-zone1-ncn-s002-mgujxl
+        /usr/bin/podman: stderr Error: no such container ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw-site1-zone1-ncn-s002-mgujxl
+        Non-zero exit code 125 from /usr/bin/podman container inspect --format {{.State.Status}} ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw.site1.zone1.ncn-s002.mgujxl
+        /usr/bin/podman: stderr Error: no such container ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw.site1.zone1.ncn-s002.mgujxl
+        Deploy daemon rgw.site1.zone1.ncn-s002.mgujxl ...
+        Verifying port 8080 ...
+        Cannot bind to IP 0.0.0.0 port 8080: [Errno 98] Address already in use
+        ERROR: TCP Port(s) '8080' required for rgw already in use
+        2024-09-20T19:04:56.410687+0000 mgr.ncn-s002.mneqjy [INF] Deploying daemon rgw.site1.zone1.ncn-s001.rdtynd on ncn-s001
+        2024-09-20T19:04:57.536533+0000 mgr.ncn-s002.mneqjy [INF] Removing key for client.rgw.site1.zone1.ncn-s001.rdtynd
+        2024-09-20T19:04:57.576404+0000 mgr.ncn-s002.mneqjy [ERR] Failed while placing rgw.site1.zone1.ncn-s001.rdtynd on ncn-s001: cephadm exited with an error code: 1, stderr: Non-zero exit code 125 from /usr/bin/podman container inspect --format {{.State.Status}} ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw-site1-zone1-ncn-s001-rdtynd
+        /usr/bin/podman: stderr Error: no such container ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw-site1-zone1-ncn-s001-rdtynd
+        Non-zero exit code 125 from /usr/bin/podman container inspect --format {{.State.Status}} ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw.site1.zone1.ncn-s001.rdtynd
+        /usr/bin/podman: stderr Error: no such container ceph-56cdd77c-7184-11ef-9f67-42010afc0104-rgw.site1.zone1.ncn-s001.rdtynd
+        Deploy daemon rgw.site1.zone1.ncn-s001.rdtynd ...
+        Verifying port 8080 ...
+        Cannot bind to IP 0.0.0.0 port 8080: [Errno 98] Address already in use
+        ERROR: TCP Port(s) '8080' required for rgw already in use
+        ```
+
+        **Note** the name of the daemon attempting to deploy for the next step, in this case `rgw.site1.zone1`
+
+        **Also Note** that `rgw.site1.zone1` should no longer be used and instead replaced with `rgw.site1` instead to avoid conflicting deployments.
```

Comment:
```suggestion
```
Move this comment to the next section, I think it is more clear to have it when removing one of the deployments.